### PR TITLE
Internal Datasets for Examples and Examples Added

### PR DESCRIPTION
**Pull request by ChloeYou**
- created two internal datasets, one for epi_df and one for epi_archive, and added documentation for them in data.R
- created examples for functions in slide.R, correlation.R, archive.R using the internal datasets
- attributions added to documentation of internal data in data.R re: cmu-delphi/epiprocess#73. please let me know if this is a good place to put the attribution. 

Thank you!

### CODE REVIEW

File: R/archive.R [hunk 1]
```diff
@@ -448,6 +448,24 @@ epi_archive =
...
+#' covidcast(data_source = "jhu-csse",
+#' signals = "confirmed_7dav_incidence_prop",
+#' time_type = "day",
+#' geo_type = "state",
+#' time_value = epirange(20200601, 20200603),
+#' geo_values = "ca,fl",
+#' issues = epirange(20200601, 20200603)
+#' ) %>%
+#' fetch_tbl() %>%
+#' select(geo_value, time_value, version = issue, case_rate = value) %>%
```
Comment (dajmcdon) [inline R/archive.R]: This part is all too complicated I think. Can you maybe just generate a meaningless data frame and convert that (perhaps with and without some extra keys)?

File: R/correlation.R [hunk 2]
```diff
@@ -40,6 +40,33 @@
...
+#'         var1 = case_rate_7d_av, 
+#'         var2 = death_rate_7d_av, 
+#'         cor_by = geo_value)
+#' 
+#' # correlation grouped by location and incorporates lagged cases rates
+#' epi_cor(x = jhu_csse_daily, 
+#'         var1 = case_rate_7d_av, 
+#'         var2 = death_rate_7d_av, 
+#'         cor_by = geo_value, 
+#'         dt1 = -10)
```
Comment (dajmcdon) [inline R/correlation.R:69]: Good.

File: R/data.R [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+#' This example data ranges from June 1 to June 15, 2020. 
+#'
+#' @format An `epi_archive` data format. The data table DT has 160 rows and 5 columns:
+#' \describe{
+#'   \item{geo_value}{the geographic value associated with each row of measurements.}
+#'   \item{time_value}{the time value associated with each row of measurements.}
+#'   \item{version}{ the time value specifying the version for each row of measurements. }
+#'   \item{percent_cli}{percentage of doctor’s visits with CLI (COVID-like illness) computed from medical insurance claims}
+#'   \item{case_rate}{7-day average signal of number of new confirmed deaths due to COVID-19 per 100,000 population, daily}
+#'   ...
```
Comment (dajmcdon) [inline R/data.R]: Is this a place holder?

File: R/data.R [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+#' }
+#' @source These data sources are provided under the terms of the 
+#' \href{https://creativecommons.org/licenses/by/4.0/}{Creative Commons Attribution license:}
+#' * \href{https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/doctor-visits.html}{Doctor Visits}
+#' * \href{https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/hospital-admissions.html}{Hospital Admissions}
+#' * \href{https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/indicator-combination-inactive.html}{Indicator Combination:} signals with names beginning `nmf_`
+#' * \href{https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/quidel.html}{Quidel}
+#' * \href{https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/fb-survey.html}{COVID-19 Trends and Impact Survey}
```
Comment (dajmcdon) [inline R/data.R]: I suspect we don't need these since we aren't including data from these sources?

File: R/slide.R [hunk 5]
```diff
@@ -88,6 +88,13 @@
 #' @importFrom lubridate days weeks
 #' @importFrom rlang .data .env !! enquo enquos sym
 #' @export
+#' @examples 
+#' library(dplyr)
+#' 
+#' # slide a 7-day trailing average formula on cases
```
Comment (dajmcdon) [inline R/slide.R]: This is fine, but maybe include some other examples using the `align` and `as_list_col` arguments?

**Review by dajmcdon — CHANGES_REQUESTED** [review]

File: R/data.R (hunk 3 above)
Comment (ChloeYou) [reply R/data.R]: ah yes good catch, I'll remove this.

File: R/data.R (hunk 4 above)
Comment (ChloeYou) [reply R/data.R]: as shown here: https://cmu-delphi.github.io/delphi-epidata/api/covidcast_licensing.html#creative-commons-attribution

Guess we only need the doctor visit one then

File: R/slide.R (hunk 5 above)
Comment (ChloeYou) [reply R/slide.R]: sounds good. I'll add those as well.

File: R/methods-epi_archive.R [hunk 6]
```diff
@@ -67,6 +70,18 @@ epix_as_of = function(x, max_version, min_time_value = -Inf) {
 #'   ```
 #' 
 #' @export
+#' @examples 
+#' library(dplyr)
```
Comment (dajmcdon) [inline R/methods-epi_archive.R]: This is the part that you're going to adjust right?

File: R/outliers.R [hunk 7]
```diff
@@ -120,6 +148,16 @@ detect_outlr = function(x = seq_along(y), y,
 #'   `lower`, `upper`, and `replacement`.
 #'
 #' @export
+#' @examples 
+#' library(dplyr)
```
Comment (dajmcdon) [inline R/outliers.R]: here too

File: R/outliers.R [hunk 8]
```diff
@@ -208,6 +246,17 @@ detect_outlr_rm = function(x = seq_along(y), y, n = 21,
 #' @importFrom stats median
 #' @importFrom tidyselect starts_with
 #' @export
+#' @examples 
+#' library(dplyr)
```
Comment (dajmcdon) [inline R/outliers.R]: and here

File: R/slide.R [hunk 9]
```diff
@@ -88,6 +88,27 @@
 #' @importFrom lubridate days weeks
 #' @importFrom rlang .data .env !! enquo enquos sym
 #' @export
+#' @examples 
+#' library(dplyr)
```
Comment (dajmcdon) [inline R/slide.R]: Does it need `dplyr` at all?

**Review by dajmcdon — CHANGES_REQUESTED** [review]
I think the only thing that remains is to remove the `dplyr` stuff as much as possible.

**Review by dajmcdon — APPROVED** [review]

**Pull request merged**